.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
```

Comment:
- [ ] Please fix linter errors, here you can see the styling errors in your code 
![image](https://user-images.githubusercontent.com/74744787/211907551-a7abe684-7d74-4757-b2a0-8a24932cbf13.png)
 
for example, the first one says that on line 2 in your style.css file, you should add one space after the colon `:`
![image](https://user-images.githubusercontent.com/74744787/211908866-c000c356-4622-4a9d-a505-3a7aade4bbc0.png)

- the second one says you should add one empty line after the `}` bracket on line 3